src/include/utils/rangerrest.h

```diff
@@ -86,10 +90,9 @@ typedef struct RangerRequestJsonArgs {
 } RangerRequestJsonArgs;
 
 RangerACLResult parse_ranger_response(char *);
-json_object *create_ranger_request_json_batch(List *);
-json_object *create_ranger_request_json(char *, AclObjectKind kind, char *, List *, bool);
+json_object *create_ranger_request_json(List *);
 int call_ranger_rest(CURL_HANDLE curl_handle, const char *request);
-extern int check_privilege_from_ranger_batch(List *);
-extern int check_privilege_from_ranger(char *, AclObjectKind kind, char *, List *, bool);
+extern int check_privilege_from_ranger(List *);
+void getClientIP(char *remote_host);
```

Comment:
Since we do not call *getClientIP()* in the outside, it seems better to make it private in rangerrest.c?